packages/trader/src/Assets/Trading/Categories/contract-type-description-video.tsx

```diff
@@ -21,7 +19,6 @@ const ContractTypeDescriptionVideo = ({ selected_contract_type }: { selected_con
     const mp4_src = React.useMemo(() => getVideoSource('mp4'), [getVideoSource]);
     const webm_src = React.useMemo(() => getVideoSource('webm'), [getVideoSource]);
 
-    if (selected_contract_type !== 'accumulator') return null;
```

Comment:
This check was removed in order no to check it twice (the only place it was used is in `TradeCategoriesGIF` in `case` of 'accumulator') and to make the video component more independent and reusable.